src/app/api/card-pricing/route.ts

```
@@ -25,20 +37,48 @@ export async function GET(request: NextRequest) {
+  const forceRefresh = searchParams.get('refresh') === 'true';
 
   // Validate card ID
   if (!cardId) {
     return NextResponse.json({ error: 'Card ID is required' }, { status: 400 });
   }
 
   try {
+    // Check cache first (unless force refresh is requested)
+    if (!forceRefresh) {
+      const cacheKey = createCacheKey(CACHE_KEYS.CARD_PRICING, { id: cardId });
+      const cachedPricing = getWithExpiry<any>(cacheKey);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)